packages/contracts/package.json

```diff
@@ -53,8 +53,8 @@
   },
   "dependencies": {
     "@kroma-network/core-utils": "^0.1.0",
-    "@openzeppelin/contracts": "4.7.3",
-    "@openzeppelin/contracts-upgradeable": "4.7.3",
+    "@openzeppelin/contracts": "4.9.3",
+    "@openzeppelin/contracts-upgradeable": "4.9.3",
```

Comment:
I think there was a reason why you upgraded the version of contracts.
Please leave a comment for this!

Reply:
From 4.7.3 to 4.9.3, there have been many changes to contracts such as Timelock, governor, erc721, etc. There are also breaking changes, so we have updated to the latest release version.
(https://github.com/OpenZeppelin/openzeppelin-contracts/releases)